crates/examples/examples/add_para.rs

```diff
@@ -15,15 +15,14 @@ async fn main() -> Result<(), anyhow::Error> {
                 .with_node(|node| node.with_name("bob"))
         })
         .with_parachain(|p| {
-            p.with_id(2000)
-                .cumulus_based(true)
-                .with_collator(|n|
+            p.with_id(2000).cumulus_based(true).with_collator(
+                |n| {
                     n.with_name("collator")
                     // TODO: check how we can clean
                     .with_command("polkadot-parachain")
-                    // .with_command("test-parachain")
-                    // .with_image("docker.io/paritypr/test-parachain:c90f9713b5bc73a9620b2e72b226b4d11e018190")
-                )
+                }, // .with_command("test-parachain")
+                   // .with_image("docker.io/paritypr/test-parachain:c90f9713b5bc73a9620b2e72b226b4d11e018190")
+            )
```

Comment:
Or we remove this lines and add a _doc_ to clarify this example is to run in `native` provider, or we add the image and allow to run in multiple providers.